lib/metadata/index.js

```diff
@@ -130,13 +129,13 @@ class Metadata {
      * @param {String} partitionerName
      */
     setPartitioner(partitionerName) {
-        if (/RandomPartitioner$/.test(partitionerName)) {
+        /* if (/RandomPartitioner$/.test(partitionerName)) {
             return (this.tokenizer = new t.RandomTokenizer());
         }
         if (/ByteOrderedPartitioner$/.test(partitionerName)) {
             return (this.tokenizer = new t.ByteOrderedTokenizer());
         }
-        return (this.tokenizer = new t.Murmur3Tokenizer());
+        return (this.tokenizer = new t.Murmur3Tokenizer()); */
```

Comment:
:question: Why are you commenting this code out? What's the purpose? Shouldn't we rather delete the code altogether and provide a `deprecated` annotation with an explanation?